package.json

```diff
@@ -24,7 +24,7 @@
   "author": "wuzhao.mail@gmail.com",
   "repository": {
     "type": "git",
-    "url": "git@github.com:react-component/time-picker.git"
+    "url": "git@github.com:Getaround/time-picker.git"
```

Comment:
When publishing, it tries to also publish to GitHub pages, so this is to make it publish to our own pages instead of trying to update their and erroring